torch/csrc/jit/script/compiler.cpp

```diff
@@ -230,14 +246,21 @@ std::vector<Value*> BuiltinFunction::call(
     Method & m,
     at::ArrayRef<Value*> inputs_,
     List<Attribute> attributes,
-    size_t n_outputs) {
+    CallsiteDescriptor cd) {
   std::vector<Value*> inputs;
   if (value) inputs.push_back(value);
   inputs.insert(inputs.end(), inputs_.begin(), inputs_.end());
-  Node * n = emitBuiltinCall(loc, m, name, inputs, attributes, n_outputs);
+  // TODO: remove when we support tuple packing for builtins.
+  if (cd.allow_varargs && cd.n_outputs == 1) {
+    cd.allow_varargs = false;
+  }
+  Node * n = emitBuiltinCall(loc, m, name, inputs, attributes, cd);
   if (!hasTensorOp(n)) {
     throw ErrorReport(loc) << "unknown builtin op";
   }
+  if (cd.n_outputs != 1 && cd.allow_varargs) {
+    throw ErrorReport(loc) << "Varargs outputs are not currently supported for builtins.";
```

Comment:
Wait what? Calling outputs varargs is really confusing, since it's a shorthand for "variable arguments"